### ISSUE TITLE
Inconsistent Validation results during training and by using allennlp evaluate

### ISSUE THREAD

**Issue by shmahaj** [allenai/allennlp#5702]
I am fine-tuning a model and the validation accuracy I get during training time using allennlp train(85%) is higher than what I get when I run allennlp evaluate(42%) on the saved model and the validation set. Could you please suggest what could be going wrong here?
